cli/src/main/java/org/openapitools/openapistylevalidator/cli/OptionManager.java

```diff
@@ -79,8 +79,10 @@ ValidatorParameters getOptionalValidatorParametersOrDefault(CommandLine commandL
                 fixConventionRenaming(jsonElement, "property");
                 Gson gson = new GsonBuilder().create();
                 parameters = gson.fromJson(jsonElement, ValidatorParameters.class);
-            } catch (Exception ignored) {
+            } catch (java.io.IOException ignored) {
                 System.out.println("Invalid path to option files, using default.");
+            } catch(com.google.gson.JsonSyntaxException e) {
```

Comment:
The code is good. Could you do a "format" or just manually add a space between `catch` and `(` ? 
Thanks!

Reply:
Nice catch.... 😛